spmd/compiler/api.py

```diff
@@ -137,6 +149,38 @@ def unwrap_local(e: object) -> object:
     return make_fx(dispatch)(tree_map(unwrap_local, args))
 
 
+def _build_dummy_add_graph(
+    dt: DTensor, node_to_obj: Dict[fx.Node, object]
+) -> fx.GraphModule:
+    """creates a graph for a dummy add function from a partial DTensor.
+    This dummy add is then used for expansion into a CommTensor expanded graph"""
```

Comment:
nit: the dummy add is used for triggering `all_reduce` on Partial DTensor during the DTensor expansion of the traced graph.

Reply:
Thanks - will update this with the next PR.